tests/integration/actions/conftest.py

```diff
@@ -72,9 +72,9 @@ def datasource(
         datasource.name == datasource_name
     ), "The datasource was not updated in the previous method call."
     yield datasource
-    context.delete_datasource(datasource_name=datasource_name)
+    context.delete_datasource(datasource_name)
```

Comment:
here is the V1 syntax:
```suggestion
    context.data_sources.delete(name=datasource_name)
```